code/__defines/items_clothing.dm

```diff
@@ -4,12 +4,40 @@
 
 #define CANDLE_LUM 3 // For how bright candles are.
 
+
```

Comment:
This file is just rearranged and comments added to explain things, nothing is actually changed.